src/styles/blocks/content.scss

```
@@ -0,0 +1,32 @@
+.content {
+  display: grid;
+
+  row-gap: 46px;
+  column-gap: 48px;
+  padding-top: 50px;
+  justify-content: center;
+}
+
+@media (min-width: 360px) {
+  .content {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
+
+@media (min-width: 488px) {
+  .content {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
```

You can rewrite like this
```suggestion
.content {
  display: grid;

  row-gap: 46px;
  column-gap: 48px;
  padding-top: 50px;
  justify-content: center;
  
  @media (min-width: 360px) {
    grid-template-columns: repeat(1, 200px);
  }
  
  ...
}
```